cli/config/config.go

```diff
@@ -26,15 +26,29 @@ const (
 var (
 	initConfigDir sync.Once
 	configDir     string
+	homeDir       string
 )
 
+// resetHomeDir is used in testing to resets the "homeDir" package variable to
+// force re-lookup of the home directory between tests.
+func resetHomeDir() {
```

Comment:
Maybe we can move this function in the tests files? I mean, tests are in the same package, so they have access to the `homeDir` variable.

Reply:
Yes, I was on the fence what would be better; I put it in the non-test file to make it more visible (I really don't like having to do this 😞)